.github/workflows/e2e_tests.yml

```diff
@@ -49,4 +49,5 @@ jobs:
       - name: Run tests
         run: |
           set -o pipefail
+          kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/release-0.52/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
```

Comment:
I think this line can instead move into `run-rollouts-manager-e2e-tests.sh`, so that it will always be present when our E2E tests run (whereas at the moment, in e2e_test.yml means it will only be when PRs are run).

Also, we probably want to avoid running `kubectl  apply -f` if the CRD is already present on the cluster (for example, if we are running E2E tests on OpenShift). Can probably do something like this in `hack/run-rollouts-manager-e2e-tests.sh`:

```bash
kubectl get crd/servicemonitors.monitoring.coreos.com
retVal=$?
if [ $retVal -ne 0 ]; then
    kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/release-0.52/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
fi
```